src/Components/TempCard/TempSort.scss

```diff
@@ -0,0 +1,65 @@
+@import "/src/mixins/grid-template.scss";
+@import "../../mixins/adaptive-rules.scss";
+
+.skeleton__total-phones {
+  height: 21px;
+  width: 80px;
+}
+
+.skeleton__dropdown {
```

Comment:
you have two files where you can see +- the same classes. Use BEm methodology like tempcard__....